src/components/Vault/VaultDepositForm.tsx

```diff
@@ -60,15 +54,10 @@ const VaultDepositForm: FC<VaultProps> = (props) => {
     address: inputTokenAddress,
   })
 
-  const amountInNumber = Number(amountIn)
-  const minAmountNumber = isNaN(amountInNumber)
-    ? 0
-    : amountInNumber - (amountInNumber * slippage) / 100
-  const minAmount = parseUnits(
-    minAmountNumber.toString(),
-    inputToken?.decimals || 18
-  )
-  const value = parseUnits(amountIn || "0", inputToken?.decimals || 18)
+  // preview redeem currently returns a value with slippage accounted for
+  // no math is required here
+  const minAmount = parseTokenUnits(amountOut, inputToken?.decimals)
```

Comment:
I don't see usePreviewDeposit sends the slippage? How does the amountOut take into account the slippage?

Reply:
@vanillaHill previewing deposit/withdraw via the API requires us to send slippage. The response is a preview of the result based on the slippage. Calculating again here will result in the incorrect value.

Reply:
yes, you're right